modules/custom/d_demo/modules/d_demo_captcha/config/install/captcha.settings.yml

```diff
@@ -0,0 +1,10 @@
+enabled_default: 0
+default_challenge: recaptcha/reCAPTCHA
+description: 'This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.'
+administration_mode: false
+allow_on_admin_pages: false
+add_captcha_description: false
+default_validation: 1
+persistence: 1
+enable_stats: false
+log_wrong_responses: false
```

Comment:
fiu fiu, tu jakieś configi, tam jakieś configi a nikt modułu nie włącza w update :thinking: to ustalałeś z @grzegorz-pietrzak-droptica żeby tak było ?

Reply:
Nie wiem, a chcemy żeby to się automatycznie włączało, skoro to rozwiązanie dla nas na demo drooplera?